src/main/java/com/iopipe/IOpipeConfiguration.java

```diff
@@ -85,6 +90,8 @@
 		cb.setInstallMethod("Disabled");
 		cb.setRemoteConnectionFactory(new NullConnectionFactory());
 		cb.setTimeOutWindow(0);
+		cb.setServiceUrl("https://localhost:80/event");
```

Comment:
I may be mistaken, but this reads like the library now defaults to sending to localhost, and then the URL is overwritten per region. I imagine we want to default to `metrics-api.iopipe.com`

Reply:
It does not, this is just the configuration that will be used if the service could not be automatically configured because the default configuration could not be initialized (likely due to a missing token). I could change the URL to something like `"service-misconfigured"` or similar. Additionally I could instead have a global default URL instead that could be always set to the default address.